typed-racket-lib/typed-racket/typecheck/tc-structs.rkt

```diff
@@ -225,9 +228,17 @@
 (define (refine-struct-variance! parsed-structs)
   (define stys (map parsed-struct-sty parsed-structs))
   (define tvarss (map (compose struct-desc-tvars parsed-struct-desc) parsed-structs))
-  (define names (map Struct-name stys))
+  (define names
+    (for/list ([parsed-struct (in-list parsed-structs)])
+      (struct-names-type-name (parsed-struct-names parsed-struct))))
   (refine-variance! names stys tvarss))
 
+;; get-struct-name : (U Struct Prefab) -> Symbol
```

Comment:
Whoops, I forgot that I added this function. I'm planning to delete it since it's unused now.